events-api/src/modules/events/events.controller.ts

```diff
@@ -47,4 +46,19 @@ export class EventsController {
       data: foundEvent,
     };
   }
+
+  @Patch(':eventId')
+  updateEvent(
+    @Param('eventId') id: string,
+    @Body()updateEventDto: CreateEventReqDto,
+  ){
+    return this.eventsService.updateEvent(id, updateEventDto)
+  }
+
+  @Delete(':eventId')
```

Comment:
Can everyone delete an event?

Reply:
Fixed: authenticated users with the role of admin can delete an event

Reply:
fixed: authenticated users with the role of `admin` can delete